modules/rostests/apitests/apphelp/apphelp.c

```diff
@@ -1017,7 +1066,8 @@ static void test_SdbGetAppPatchDir(void)
     HRESULT hr, expect_hr;
     int n;
 
-
+#if 0
+    // Newer versions seem to return garbage
```

Comment:
By that you mean, that they wouldn't throw the exception, yet set hr to something different than S_FALSE? Or, would they actually throw the exception?